changelog/unreleased/change-introduce-web-pkg

```diff
@@ -0,0 +1,6 @@
+Change: Add web-pkg package
```

Comment:
It's an enhancement (no breaking change => we call non-breaking changes `enhancement`).